functions/bobthefish_display_colors.fish

```diff
@@ -142,7 +142,11 @@ function bobthefish_display_colors -a color_scheme -d 'Print example prompt colo
   __bobthefish_finish_segments
 
   __bobthefish_start_segment $color_aws_vault
-  echo -ns aws-vault ' '
+  echo -ns aws-vault active ' '
```

Comment:
`echo -ns` here eats the space between "aws-vault" and "active" :)

Reply:
Can't believe I missed that. Fixed:

![image](https://user-images.githubusercontent.com/14284827/96386279-1f519780-11f6-11eb-8477-298d976f3952.png)